src/Organization.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"; 
+import "./Organization.css"; 
+import { Route } from "react-router-dom"; 
+
+const One = () => {
+    return(
+    <div>
+        <h1>One!</h1> 
+    </div>
+    );
+};
+
+const Two = () => {
+    return(
+        <div>
+            <h1>Two!</h1>
+        </div>
+    );
+};
+
+const Organization = (props) => {
+
+    let { org } = props.match.params;
```

Comment:
![letconst](https://user-images.githubusercontent.com/30677679/66802792-3a9f0e00-eee3-11e9-8913-d6f802ed1bdb.png)

Reply:
Done. As you can see, Karl does not write JavaScript. Karl does not know Javascript.